### DISCUSSION TITLE
构建一个obdiag 开发环境的dokcer镜像 build a docker image about obdiag dev env

### DISCUSSION THREAD

**Discussion by wayyoungboy** [oceanbase/obdiag#492] [Ideas]
计划构建一个obdiag 开发环境的dokcer镜像，降低一开始参与obdiag共建同学的学习门槛。需要内置的环境依赖如下：
1. ssh server：用于远程连接开发环境；
2. python3.8：obdiag开发环境基础；
3. obdiag原有的python依赖：较少需要额外安装的包；
4. 基于阿里云源的centos7.9：方便打包和统一环境；
5. git：用于同步代码。
.....
还有更多的想法大家可以讨论下。

We plan to build a dokcer image for the obdiag development environment in order to lower the learning curve for new students joining the obdiag co-construction project. The required built-in environment dependencies are as follows:

ssh server: for remote connections to the development environment;
python3.8: the foundation of obdiag development environment;
obdiag's original python dependencies: requiring minimal additional installation of packages;
CentOS 7.9 based on Aliyun source: for easy packaging and standardized environment;
.....
There are more ideas for discussion.

**Comment by YOMO-Lee** [comment]
assign to me

**Comment by wayyoungboy** [comment]
#493

**Comment by YOMO-Lee** [comment]
这是来自QQ邮箱的自动回复邮件。
 
您好，我最近正在休假中，无法亲自回复您的邮件。我将在假期结束后，尽快给您回复。
